staging/src/k8s.io/apiextensions-apiserver/pkg/cmd/server/testing/testserver.go

```diff
@@ -217,3 +217,21 @@ func createLocalhostListenerOnFreePort() (net.Listener, int, error) {
 
 	return ln, tcpAddr.Port, nil
 }
+
+func pkgPath() (string, error) {
+	_, thisFile, _, ok := runtime.Caller(0)
+	if !ok {
+		return "", fmt.Errorf("failed to get current file")
+	}
+
+	pkgPath := filepath.Dir(thisFile)
+	if testSrcdir, testWorkspace := os.Getenv("TEST_SRCDIR"), os.Getenv("TEST_WORKSPACE"); testSrcdir != "" && testWorkspace != "" {
+		pkgPath = filepath.Join(testSrcdir, testWorkspace, pkgPath)
```

Comment:
mind plumbing `t` here so we can `t.Logf` a message about the resolved pkgPath, and adding a comment about the environment this is enabling? (same for the other instance)